### PR TITLE
Select geoms

### PR DESCRIPTION
**Pull request by thenaomig**
The following was corrected / added:
- fixed a mis-stated assertion error in generate() that prevented it from sub-setting regions
- added a map that shows the sub-setting area selected, as a param view
- added state outlines, and county and watershed outlines within California as additional area options
- added additional error checking, and hiding scenario options that are not available for a resolution

### CODE REVIEW

**Review by bkg — COMMENTED** [review]
Thanks, just added a few comments. If could you run flake8 and black, that will take care of the style inconsistencies.

File: climakitae/selectors.py [hunk 1]
```diff
@@ -43,6 +47,55 @@
 ]
 
 # === Select ===================================
+class boundaries():
+    def __init__(self):
+        self._us_states = regionmask.defined_regions.natural_earth_v4_1_0.us_states_50
+        #this link redirect is not working:
+        #ca_counties_file = "https://data.ca.gov/dataset/e212e397-1277-4df3-8c22-40721b095f33/resource/b0007416-a325-4777-9295-368ea6b710e6/download/ca-county-boundaries.zip"
+        #so for now, load this bigger file directly from the census:
+        self._ca_counties_file = "https://www2.census.gov/geo/tiger/TIGER2020/COUNTY/tl_2020_us_county.zip"
```
[the remark this answers is not in the document]
Comment (thenaomig) [reply climakitae/selectors.py:56]: oh that's great thanks

File: climakitae/selectors.py [hunk 2]
```diff
@@ -144,25 +220,34 @@ def __init__(self):
...
 
         self._resolutions = list(set(e.metadata["nominal_resolution"] for e in self._cat.values()))
 
+        _scenario_list = []
+        for resolution in self._resolutions:
+            _temp = list(set(e.metadata["experiment_id"] for e in self._cat.values() if e.metadata["nominal_resolution"]==resolution))
+            _scenario_subset = [(self._scenario_choices[e],e) for e in _temp]
+            _scenario_subset = dict(_scenario_subset)
+            _scenario_list.append((resolution,_scenario_subset))
+        self._scenarios = dict(_scenario_list)
```
[the remark this answers is not in the document]
Comment (thenaomig) [reply climakitae/selectors.py:238]: True now, but wasn't true when all the 9km data wasn't up yet, and could be relevant again when there are 3km simulations for only some of them.

**Comment by odoherty** [comment]
Suggest to change default area selection box to 32.5 to 42 N; -125.5 to -114 W to cover California domain by default

**Pull request merged**